parser.cpp

```diff
@@ -1118,10 +1130,10 @@ namespace Sass {
   {
     Expression* conj1 = parse_conjunction();
     // if it's a singleton, return it directly; don't wrap it
-    if (!peek< sequence< kwd_or, negate< identifier > > >()) return conj1;
+    if (!peek_css< kwd_or >()) return conj1;
```

Comment:
No, but [`kwd_or`] [1] is forced to have word boundaries :wink: 

[1]: https://github.com/sass/libsass/blob/master/prelexer.cpp#L526